noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/event.nr

```diff
@@ -60,31 +63,26 @@ use std::aes128::aes128_encrypt;
 /// This particular log assembly strategy (AES 128) requires the event (and the
 /// event_type_id) to be converted into bytes, because the aes function
 /// operates on bytes; not fields.
-/// NB: The extra `+ 32` bytes is for the event_type_id:
-fn compute_event_plaintext_for_this_strategy<Event, let N: u32>(event: Event) -> [u8; N * 32 + 32]
+/// NB: The extra `+ 1` is for the event_type_id:
+fn compute_event_plaintext_for_this_strategy<Event, let N: u32>(event: Event) -> [u8; (N + 1) * 32]
 where
     Event: EventInterface<N>,
 {
     let serialized_event = Serialize::<N>::serialize(event);
 
-    let event_type_id_bytes: [u8; 32] = Event::get_event_type_id().to_field().to_be_bytes();
-
-    let mut plaintext_bytes = [0 as u8; N * 32 + 32];
-
-    for i in 0..32 {
-        plaintext_bytes[i] = event_type_id_bytes[i];
-    }
-
+    let mut fields = [0; N + 1];
+    fields[0] = Event::get_event_type_id().to_field();
     for i in 0..serialized_event.len() {
-        let bytes: [u8; 32] = serialized_event[i].to_be_bytes();
-        for j in 0..32 {
-            plaintext_bytes[32 + i * 32 + j] = bytes[j];
-        }
+        fields[i + 1] = serialized_event[i];
     }
 
-    plaintext_bytes
+    fields_to_be_bytes_32(fields)
```

Comment:
Did this change here and in the note.nr because when decrypting we use be_bytes_32_to_fields. And those 2 functions are guaranteed to match.